test/Text/CssSpec.hs

```diff
@@ -278,8 +278,8 @@ bin {
     it "lucius media" $ do
       celper "@media only screen{foo bar{baz:bin}}" $(luciusFile "test/cassiuses/external-media.lucius")
       celper "@media only screen {\n    foo bar {\n        baz: bin;\n    }\n}\n" $(luciusFileDebug "test/cassiuses/external-media.lucius")
-      celper "@media only screen{foo bar{baz:bin}}" [lucius|
-        @media only screen{
+      celper "@media only screen {foo bar{baz:bin}}" [lucius|
```

Comment:
I'd prefer not to see unrelated changes in this PR. This line specifically is present to test that things work both with and without extra spacing.